content/en/docs/concepts/security/pod-security-admission.md

```diff
@@ -77,53 +82,63 @@ pod-security.kubernetes.io/<MODE>: <LEVEL>
 # version that shipped with a given Kubernetes minor version (e.g. v{{< skew latestVersion >}}).
 #
 # MODE must be one of `enforce`, `audit`, or `warn`.
-# VERSION must be a valid Kubernetes version label.
+# VERSION must be a valid Kubernetes minor version, or `latest`.
 pod-security.kubernetes.io/<MODE>-version: <VERSION>
 ```
 
 Check out [Enforce Pod Security Standards with Namespace Labels](/docs/tasks/configure-pod-container/enforce-standards-namespace-labels) to see example usage.
 
+## Pod Templates and Pod Controllers
+
+Pods are often created indirectly, by creating a pod {{< glossary_tooltip term_id="controller" >}}
+object such as a {{< glossary_tooltip term_id="deployment" >}} or {{< glossary_tooltip term_id="job">}}.
+To help catch violations early, both the audit and warning modes are applied to the pod
+controller objects. However, enforce mode is **not** applied to pod controller objects, only to the
+resulting pod object.
+
+[pod controllers]: #pod-templates-and-pod-controllers
```

Comment:
same above, pls use inline links when possible.
This flavor of links are difficult for the link-checker script to parse.